core/src/main/java/com/microsoft/applicationinsights/internal/profiler/ProfilerServiceInitializer.java

```diff
@@ -162,19 +163,21 @@ static UploadCompleteHandler sendServiceProfilerIndex(TelemetryClient telemetryC
             event.getProperties().putAll(done.getServiceProfilerIndex().getProperties());
             event.getMetrics().putAll(done.getServiceProfilerIndex().getMetrics());
             telemetryClient.track(event);
+            telemetryClient.track(new TraceTelemetry("StopProfiler succeeded."));
```

Comment:
can you add a comment here that this is a very specific message that the backend looks for (as opposed to a diagnostic message for the user)?